src/components/Identity/processIdSyncs.js

```diff
@@ -11,21 +11,24 @@ governing permissions and limitations under the License.
 */
 
 import { assign, cookie, fireDestinations } from "../../utils";
+import namespace from "../../constants/namespace";
 
 const millisecondsPerHour = 60 * 60 * 1000;
 
 const getControlObject = () => {
-  return JSON.parse(cookie.get("adobeIdSyncControl") || "{}");
+  return JSON.parse(cookie.get(`${namespace}idSyncControl`) || "{}");
```

Comment:
Let's add a TODO to use the `Alloy` cookie [PR26](https://github.com/adobe/alloy/pull/26) is merged.